spec/order_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'order'
+
+describe Order do
+  subject(:order) {described_class.new}
+  let(:menu_class) {double :menu_class, new: menu}
+  let(:menu) {double :menu}
+  let(:dish) {double :dish, to_sym: :dish}
+
+  context '#select item (when dish is on the menu)' do
+    before do
+      allow(order).to receive(:check_menu).and_return(true)
```

Comment:
Looks like you stubbed out the subject of your tests instead of injecting the menu double.